src/xenia/app/emulator_window.h

```diff
@@ -54,6 +55,14 @@ class EmulatorWindow {
   void GpuTraceFrame();
   void GpuClearCaches();
   void ShowHelpWebsite();
+  void UserlangEnglish();  
```

Comment:
Don't add separate function calls per language - instead define a function like SetLanguage(Language l) (where Language is an enum of values)